kie-spring-boot/kie-spring-boot-autoconfiguration/jbpm-spring-boot-autoconfiguration/src/main/java/org/jbpm/springboot/services/SpringKModuleDeploymentService.java

```diff
@@ -50,7 +53,16 @@ protected RegisterableItemsFactory getRegisterableItemsFactory(AuditEventBuilder
         return factory;
     }
 
-    
+    protected Map<String, Object> buildContextParameters(KieContainer kieContainer) {
+        Map<String, Object> contextParams = new HashMap<>();
+        Map<String, EntityManagerFactory> emfsFound = context.getBeansOfType(EntityManagerFactory.class);
+        for(Map.Entry<String, EntityManagerFactory> entry : emfsFound.entrySet()) {
+            contextParams.put(entry.getKey(), entry.getValue());
+        }
+        contextParams.putAll(super.buildContextParameters(kieContainer));
```

Comment:
In the parent invocation, ("entityManagerFactory", emf) is also added, shouldn't it be redundant?

Reply:
not really. 
Not all deployment service use this spring for instance kie server. so the parents needs to add the entity manager.
Also we need the parameters defined in there.
There is no point to do any check away just to remove the jbpm entity manager as it cannot be override.